modules/allowances/package.json

```diff
@@ -52,6 +60,10 @@
     "solidity-coverage": "^0.8.14",
     "ts-node": "^10.9.2",
     "typechain": "^8.3.2",
-    "typescript": "^5.7.2"
+    "typescript": "^5.7.2",
+    "zksync-ethers": "^6.16.2"
+  },
+  "dependencies": {
+    "patch-package": "^8.0.0"
```

Comment:
where is this dependency used?

Reply:
good catch. Its unused now. I was using it to patch dependencies initially